CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe CLI package will be documented in this file.
 
+## Recent Changes
+- Enhancement: Addedoptions like `--lrecl, --recfm` to zos-files create options. [788](https://github.com/zowe/zowe-cli/issues/788)
```

Comment:
Minor change, could you change: "Addedoptions" to "Added options" please?

Reply:
Thank you @Pranay154 , this is corrected now.